cmd/ltctl/comparison.go

```diff
@@ -172,6 +173,33 @@ func RunComparisonCmdF(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func CollectComparisonCmdF(cmd *cobra.Command, args []string) error {
+	deployerConfig, err := getConfig(cmd)
+	if err != nil {
+		return err
+	}
+
+	configFilePath, _ := cmd.Flags().GetString("comparison-config")
+	cfg, err := comparison.ReadConfig(configFilePath)
+	if err != nil {
+		return fmt.Errorf("failed to read comparison config: %w", err)
+	}
+
+	cmp, err := comparison.New(cfg, &deployerConfig)
+	if err != nil {
+		return fmt.Errorf("failed to initialize comparison object: %w", err)
+	}
+
+	merr := merror.New()
+	for _, id := range cmp.GetDeploymentIds() {
+		if err := collect(deployerConfig, id, id+"_"); err != nil {
+			merr.Append(err)
+		}
+	}
```

Comment:
Is there an opportunity here to use some parallelization?

Reply:
There is, but I'd rather keep this as simple as possible, as the potential gain in this would be in the realm of a few seconds, which doesn't seem worth it at the moment. I'm 1/5 though, if you have a stronger opinion I can parallelize it.

Reply:
I asked because I vaguely remember this taking more than a few seconds and multiplied in a big deployment could become several minutes. But it's the sort of improvement we can do at a later point when someone will feel the pain of waiting :p